sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionFeeSchedule.java

```diff
@@ -0,0 +1,61 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import javax.annotation.Nullable;
+
+public class TransactionFeeSchedule {
+    private RequestType requestType;
+    @Nullable
+    private FeeData feeData;
+
+    public TransactionFeeSchedule() {
+        requestType = RequestType.NONE;
+        feeData = null;
+    }
+
+    RequestType getRequestType() {
+        return requestType;
+    }
+    boolean hasFeeData() {
+        return feeData != null;
+    }
+    @Nullable
+    FeeData getFeeData() {
+        assert hasFeeData();
+        return feeData;
+    }
+
+    TransactionFeeSchedule setRequestType(RequestType requestType) {
+        this.requestType = requestType;
+        return this;
+    }
+    TransactionFeeSchedule setFeeData(@Nullable FeeData feeData)
+    {
+        this.feeData = feeData;
+        return this;
+    }
+
+    static TransactionFeeSchedule fromProtobuf(com.hedera.hashgraph.sdk.proto.TransactionFeeSchedule transactionFeeSchedule) {
+        return new TransactionFeeSchedule()
+            .setRequestType(RequestType.valueOf(transactionFeeSchedule.getHederaFunctionality()))
+            .setFeeData(transactionFeeSchedule.hasFeeData() ? FeeData.fromProtobuf(transactionFeeSchedule.getFeeData()) : null);
+    }
+    public static TransactionFeeSchedule fromBytes(byte[] bytes) throws InvalidProtocolBufferException {
+        return fromProtobuf(com.hedera.hashgraph.sdk.proto.TransactionFeeSchedule.parseFrom(bytes).toBuilder().build());
+    }
+
+
+
+    com.hedera.hashgraph.sdk.proto.TransactionFeeSchedule toProtobuf()
+    {
```

Comment:
Curly brace should be on the previous line.

Reply:
Curly braces on the next line is my "native" indentation style.  Old habits die hard.  I tried to keep to the correct style, but I kept hitting the enter key before typing the { without thinking about it.  I had to go back and undo it at LEAST half a dozen times.  Guess I still missed a few.  =P